test/IISExpress.FunctionalTests/InProcess/StartupTests.cs

```diff
@@ -59,7 +59,7 @@ public async Task InvalidProcessPath_ExpectServerError(string path, string argum
 
             StopServer();
 
-            EventLogHelpers.VerifyEventLogEvent(deploymentResult, TestSink, $@"Application '{Regex.Escape(deploymentResult.ContentRoot)}\\' wasn't able to start. {subError}");
+            EventLogHelpers.VerifyEventLogEvent(deploymentResult, $@"Application '{Regex.Escape(deploymentResult.ContentRoot)}\\' wasn't able to start. {subError}");
```

Comment:
Replace Regex.Escape with EscapedContentRoot and make EscapedContentRoot a helper.